pocs/utils/rs232.py

```diff
@@ -51,33 +51,38 @@ class SerialData(object):
     .. doctest::
 
         >>> import serial
+
         # Register our serial simulators
         >>> serial.protocol_handler_packages.append('pocs.tests.serial_handlers')
+
         # Create a fake device
         >>> from pocs.tests.serial_handlers.protocol_buffers import SetRBufferValue as WriteFakeDevice
-        >>> from pocs.tests.serial_handlers.protocol_buffers import GetWBuffer as ReadFakeDevice
+        >>> from pocs.tests.serial_handlers.protocol_buffers import GetWBufferValue as ReadFakeDevice
+        >>> from pocs.tests.serial_handlers.protocol_buffers import ResetBuffers
 
         # Import our serial utils
         >>> from pocs.utils.rs232 import SerialData
 
         # Connect to our fake buffered device
         >>> device_listener = SerialData(port='buffers://')
+        >>> ResetBuffers()
```

Comment:
Perhaps add a note that only a single SerialData instance should open 'buffers://' at any one time because it is a very limited test fixture.